screen/send/SendDetails.tsx

```
@@ -537,506 +1001,136 @@
 
-  const handleRemoveRecipient = () => {
-    if (addresses.length > 1) {
-      const newAddresses = [...addresses];
-      newAddresses.splice(scrollIndex.current, 1);
+    const targetsOrig = JSON.parse(JSON.stringify(targets));
+    // preserving original since it will be mutated
 
-      // Adjust the current index if the last item was removed
-      const newIndex = scrollIndex.current >= newAddresses.length ? newAddresses.length - 1 : scrollIndex.current;
+    // without forcing `HDSegwitBech32Wallet` i had a weird ts error, complaining about last argument (fp)
+    const { tx, outputs, psbt, fee } = (wallet as HDSegwitBech32Wallet)?.createTransaction(
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)